upcloud_api/storage.py

```diff
@@ -55,12 +56,26 @@ def _reset(self, **kwargs) -> None:
         elif 'storage_size' in kwargs:
             self.size = kwargs['storage_size']
 
+        if 'encrypted' in kwargs and kwargs['encrypted'] == 'yes':
```

Comment:
Style/nit: could simplify to 
```suggestion
        if kwargs.get('encrypted') == 'yes':
```

Reply:
:+1: